docs/Developers/Newcomers_Guide.rst

```diff
@@ -196,6 +196,8 @@ After that, you can run coala by simply typing
 
 into your bash. This will analyze your code and help you fix it.
 
+After making the changes to test the code CI tools are used in coala repositories. To add CI tools to your forked repository, check out this `Adding CI to you fork <http://api.coala.io/en/latest/Developers/Adding_CI.html>`_
```

Comment:
Try making the message a little better. This is just a suggestion.
"After making the changes, you can test the code, to your forked repository by adding the required CI tools. See [here](http://api.coala.io/en/latest/Developers/Adding_CI.html)"